samples/mappings/Microsoft/Common/DotNetRuntime.packagemap.json

```diff
@@ -0,0 +1,1870 @@
+          {
+            "version": "9.*",
+            "prerelease": true
+          }
+        ],
+        ".NETFramework,Version=v4.5": [
+          {
+            "version": "9.*",
+            "prerelease": true
+          }
+        ],
+        ".NETStandard,Version=v2.0": [
+          {
+            "version": "",
+            "prerelease": true
+          }
+        ]
+      }
+    },
+    {
+      "name": "Microsoft.Bcl.HashCode",
+      "frameworks": {
+        ".NETCoreApp,Version=v8.0": [
+          {
+            "version": ""
+          }
+        ],
+        ".NETFramework,Version=v4.5": [
+          {
+            "version": ""
+          }
+        ],
+        ".NETStandard,Version=v2.0": [
+          {
+            "version": ""
+          }
+        ],
+        ".NETStandard,Version=v2.1": []
+      }
+    },
+    {
+      "name": "Microsoft.Bcl.Numerics",
+      "frameworks": {
+        ".NETCoreApp,Version=v8.0": [
+          {
+            "version": "8.*"
+          }
+        ],
+        ".NETCoreApp,Version=v9.0": [
+          {
+            "version": "9.*",
+            "prerelease": true
+          }
+        ],
+        ".NETStandard,Version=v2.0": [
+          {
+            "version": ""
+          }
+        ]
+      }
+    },
+    {
+      "name": "Microsoft.Bcl.TimeProvider",
+      "frameworks": {
+        ".NETCoreApp,Version=v8.0": [
+          {
+            "version": "8.*"
+          }
+        ],
+        ".NETCoreApp,Version=v9.0": [
+          {
+            "version": "9.*",
+            "prerelease": true
+          }
+        ],
+        ".NETStandard,Version=v2.0": [
+          {
+            "version": ""
+          }
+        ]
+      }
+    },
+    {
+      "name": "Microsoft.CSharp",
+      "frameworks": {
+        ".NETStandard,Version=v2.0": [
+          {
+            "version": ""
+          }
+        ]
+      }
+    },
+    {
+      "name": "Microsoft.Diagnostics.Tracing.EventSource.Redist",
+      "frameworks": {
+        ".NETCoreApp,Version=v6.0": [
+          {
+            "version": "6.*"
+          }
+        ],
+        ".NETCoreApp,Version=v7.0": [],
+        ".NETCoreApp,Version=v8.0": [],
+        ".NETFramework,Version=v4.5": [
+          {
+            "version": ""
+          }
+        ],
+        ".NETStandard,Version=v2.0": []
+      }
+    },
+    {
+      "name": "Microsoft.DotNet.ILCompiler",
+      "frameworks": {
+        ".NETCoreApp,Version=v7.0": [
+          {
+            "version": "7.*"
+          }
+        ],
+        ".NETCoreApp,Version=v8.0": [
+          {
+            "version": "8.*"
+          }
+        ],
+        ".NETCoreApp,Version=v9.0": [
+          {
+            "version": "9.*",
+            "prerelease": true
+          }
+        ],
+        ".NETStandard,Version=v2.0": [
+          {
+            "version": ""
+          }
+        ]
+      }
+    },
+    {
+      "name": "Microsoft.Extensions.Caching.Abstractions"
```

Comment:
>Extensions [](http://example.com/codeflow?start=25&length=10)

why all maps are in samples? Samples should have just some meaningful example but not everything. If we change any format we would have to process soo much text in samples

Reply:
if you wanted to let public users know what are default maps, maybe we should make it "official" and put under docs\maps\defaults etc.. Though they would be out of sync sometimes if we don't keep them up to date every time.

Reply:
I was mostly aiming to provide a sufficient breadth of samples for vendors to use as a reference. I'm ok with removing most of what I have there now.

Reply:
I've removed the maps and replaced them with small generic samples (see Sample.apimap.json).